celestia/chain.json

```diff
@@ -83,6 +80,16 @@
           "type": "go",
           "version": "v6.2.1"
         }
+      },
+      {
+        "name": "v2",
+        "recommended_version": "v2.1.2",
+        "compatible_versions": [
+          "v2.0.0",
+          "v2.1.0",
+          "v2.1.1",
+          "v2.1.2"
+        ]
       }
     ],
     "sdk": {
```

Comment:
Please update the sdk version here to match the changes to codebase.

Reply:
Resolved via ad5cf851c8945dcce2c319ff5f3e351bee108824